### DISCUSSION TITLE
How would you go about having a param act as a folder route?

### DISCUSSION THREAD

**Discussion by byOskar** [posva/unplugin-vue-router#52] [Questions and Help] [answered]
e.g. /orders/:id/view

**Comment by byOskar** [comment, marked as answer]
I see in the gif at the top of your README, you'd have the orders folder and within that you would have the file [id].view.vue

**Comment by posva** [comment]
you can also have a folder named `[id]`

**Reply by byOskar** [reply]
ah great, keen to use this router!

**Reply by posva** [reply]
It's still vue-router 😄
